model/Security/Properties/vectorString.md

```diff
@@ -17,7 +17,8 @@ Supports vectorStrings specified in all CVSS versions.
 
 String values for the vectorString range must only include the abbreviated form
 of metric names specified in CVSS specifications, e.g.
-[https://www.first.org/cvss/v4.0/specification-document#Vector-String](https://www.first.org/cvss/v4.0/specification-document#Vector-String)
+[Table 23: Base, Threat and Environmental Vectors](https://www.first.org/cvss/v4.0/specification-document#Vector-String)
```

Comment:
I would remove the "Table: 23" since that is local to the Common Vulnerability Scoring System version 4.0: Specification Document.  Alternatively, we could prefix Table 23 with "Common Vulnerability Scoring System version 4.0: Specification Document", however that gets rather long.

Reply:
Thank you. I have removed it.

Reply:
Oh. Actually. The full sentence is

> e.g. [Table 23: Base, Threat and Environmental Vectors](https://www.first.org/cvss/v4.0/specification-document#Vector-String) of Common Vulnerability Scoring System version 4.0: Specification Document.

Reply:
It is now read

> String values for the vectorString range must only include the abbreviated form of metric names specified in CVSS specifications, e.g. [Common Vulnerability Scoring System Vector String](https://www.first.org/cvss/v4.0/specification-document#Vector-String).